res/TensorFlowLiteRecipes/Net_RoPE_000/test.recipe

```diff
@@ -0,0 +1,211 @@
+operand {
```

Comment:
the rope part was extracted from the tg model(layers.tg.dynamic.1.q.opt.circle) and I created tflite receipe by referring to the graph.
and, the sin/cos table was divided into 0~80 in the range of -1 to 1, made into constants, and added as input.
and, the tg model is a quantized model and the data type is INT16. I tried to create a receipt with this data type, but
an error occurred because the luci interpreter's handling of INT16 was abnormal, so I created a receipe with Float32.
i think modifying the interpreter to support INT16 should be classified as a different task, and is it necessary? I think we need to check first.
and, through luci-pass-value-test, It was confirmed that the tflite model and circle model fused with rope op have the same results.